test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxserviceTests.cs

```diff
@@ -436,6 +436,92 @@ public async void GenerateDeployPlan()
             }
         }
 
+        // <summary>
+        /// Verify the generate deploy plan request
+        /// </summary>
+        [Fact]
+        public async void DeployWithSqlCmdVariables()
```

Comment:
Wasn't sure whether the preference would be for separate tests or to refactor the existing Deploy and GenerateScript tests to also check sqlcmdvar support.  Either way's fine by me.